json_schemas/framework-agreement-details.json

```diff
@@ -15,6 +15,57 @@
       "minLength": 1,
       "type": "string"
     },
+    "frameworkRefDate": {
+      "minLength": 1,
+      "type": "string",
+      "patterm": "^[0-9-]+$"
+    },
+    "frameworkStartDate": {
+      "minLength": 1,
+      "type": "string",
+      "patterm": "^[0-9/]+$"
+    },
+    "frameworkEndDate": {
+      "minLength": 1,
+      "type": "string",
+      "patterm": "^[0-9/]+$"
```

Comment:
FWIW, I think the highest priority would be to avoid duplication (opportunity for inconsistency) of data. Ideally, all these dates would be in some non-ambiguous machine-readable format, formatted for display on the front end, and 'human readable date strings' in the framework data would _not_ be a thing.